align/cell_fabric/gen_lef.py

```diff
@@ -2,7 +2,7 @@
 import json
 import collections
 
-def lef_from_layout_d(layout_d, fp, out_lef, cell_pin, bodyswitch, blockM, *, exclude_layers, Scale_factor, m1pitch, m2pitch):
+def lef_from_layout_d(layout_d, fp, out_lef, bodyswitch, blockM, *, exclude_layers, Scale_factor, m1pitch, m2pitch):
```

Comment:
@soneryaldiz It seems that we don't need to pass the list to the lef generator anymore. This also means that we don't need to generate it during primitive construction. (see below.)